templates/mvcviewcomponent.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNet.Mvc;
+
+namespace <%= namespace %>
+{
+    public class <%= classname %> : ViewComponent
+    {
+		public IViewComponentResult Invoke()
```

Comment:
Bah, sorry - I actually hacked it together with Notepad2 - thought I had fixed the indentation :-/